submit/controllers/__init__.py

```diff
@@ -1,4 +1,8 @@
 from .verify_user import verify_user
 from .authorship import authorship
+from .license import license
```

Comment:
``license`` is a built-in...but a fairly useless one for our purposes, so probably not a big deal.